src/metabase/request/core.clj

```diff
@@ -0,0 +1,53 @@
+(ns metabase.request.core
+  "API namespace for stuff related to Ring (HTTP) requests, such as info associated with the current request."
+  (:require
+   [metabase.request.cookies]
+   [metabase.request.current]
+   [metabase.request.session]
+   [metabase.request.util]
+   [potemkin :as p]))
```

Comment:
Please use `metabase.util.namespaces/import-fns` just for uniformity. It expands to `p/import-vars` in CLJ, so you still get `doc` and go-to-definition just like Potemkin.

Reply:
I'll open a GH issue to fix all of the usages